--[[

	‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēó‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēó ‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēó‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēó   ‚Ėą‚Ėą‚ēó     ‚Ėą‚Ėą‚ēó   ‚Ėą‚Ėą‚ēó ‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēó 
	‚Ėą‚Ėą‚ēĒ‚ēź‚ēź‚ēź‚ēź‚ēĚ‚Ėą‚Ėą‚ēĒ‚ēź‚ēź‚Ėą‚Ėą‚ēó‚Ėą‚Ėą‚ēĒ‚ēź‚ēź‚ēź‚ēź‚ēĚ‚Ėą‚Ėą‚ēĒ‚ēź‚ēź‚ēź‚ēź‚ēĚ   ‚Ėą‚Ėą‚ēĎ     ‚Ėą‚Ėą‚ēĎ   ‚Ėą‚Ėą‚ēĎ‚Ėą‚Ėą‚ēĒ‚ēź‚ēź‚Ėą‚Ėą‚ēó
	‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēó  ‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēĒ‚ēĚ‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēó  ‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēó   ‚Ėą‚Ėą‚ēĎ     ‚Ėą‚Ėą‚ēĎ   ‚Ėą‚Ėą‚ēĎ‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēĎ
	‚Ėą‚Ėą‚ēĒ‚ēź‚ēź‚ēĚ  ‚Ėą‚Ėą‚ēĒ‚ēź‚ēź‚ēź‚ēĚ ‚Ėą‚Ėą‚ēĒ‚ēź‚ēź‚ēĚ  ‚ēö‚ēź‚ēź‚ēź‚ēź‚Ėą‚Ėą‚ēĎ   ‚Ėą‚Ėą‚ēĎ     ‚Ėą‚Ėą‚ēĎ   ‚Ėą‚Ėą‚ēĎ‚Ėą‚Ėą‚ēĒ‚ēź‚ēź‚Ėą‚Ėą‚ēĎ
	‚Ėą‚Ėą‚ēĎ     ‚Ėą‚Ėą‚ēĎ     ‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēó‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēĎ‚Ėą‚Ėą‚ēó‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēó‚ēö‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚Ėą‚ēĒ‚ēĚ‚Ėą‚Ėą‚ēĎ  ‚Ėą‚Ėą‚ēĎ
	‚ēö‚ēź‚ēĚ     ‚ēö‚ēź‚ēĚ     ‚ēö‚ēź‚ēź‚ēź‚ēź‚ēź‚ēź‚ēĚ‚ēö‚ēź‚ēź‚ēź‚ēź‚ēź‚ēź‚ēĚ‚ēö‚ēź‚ēĚ‚ēö‚ēź‚ēź‚ēź‚ēź‚ēź‚ēź‚ēĚ ‚ēö‚ēź‚ēź‚ēź‚ēź‚ēź‚ēĚ ‚ēö‚ēź‚ēĚ  ‚ēö‚ēź‚ēĚ

	Project: FPES.lua
	Created by Monia
	Contributor: MaxlaserTech

	Note: Solara and Xeno sucks
--]]

return {
	["QuickTeleport"] = {
		["Unreliable"] = false,
		["ClientEvent"] = {
			["Id"] = "",
			["Fire"] = function() end,
			["On"] = function() end,
			["Unreliable"] = false,
		},
		["Id"] = "",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["HalloweenCandy"] = {
		["Unreliable"] = false,
		["Id"] = "	",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["Misc3"] = {
		["StatUpdate"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "S",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "S",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["CamPic"] = {
			["Unreliable"] = false,
			["Id"] = "W",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["PlaySound"] = {
			["Unreliable"] = false,
			["Id"] = "V",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Flicker"] = {
			["Unreliable"] = false,
			["Id"] = "Z",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["DistanceShake"] = {
			["Unreliable"] = false,
			["Id"] = "[",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["MenuTrigger"] = {
			["Unreliable"] = false,
			["Id"] = "U",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["CameraCooldown"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "Y",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "Y",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["TogglePrompts"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "T",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "T",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Blind"] = {
			["Unreliable"] = false,
			["Id"] = "X",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
	},
	["DeltaShardsChanged"] = {
		["Unreliable"] = false,
		["Id"] = "",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["Misc2"] = {
		["RoundSummary"] = {
			["Unreliable"] = false,
			["Id"] = "L",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["FinishedTest"] = {
			["Unreliable"] = false,
			["Id"] = "P",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["ResetButtonEnabled"] = {
			["Unreliable"] = false,
			["Id"] = "M",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["DisplaySystemMsg"] = {
			["Unreliable"] = false,
			["Id"] = "O",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["CycleTrigger"] = {
			["Unreliable"] = false,
			["Id"] = "K",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Clock"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "N",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "N",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["StartTest"] = {
			["Unreliable"] = false,
			["Id"] = "Q",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["SendNotification"] = {
			["Unreliable"] = false,
			["Id"] = "R",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
	},
	["GamemodeJoinRequest"] = {
		["Unreliable"] = false,
		["Id"] = "`",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["BlockRequest"] = {
		["Unreliable"] = false,
		["Id"] = "_",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["FootstepSFX"] = {
		["Unreliable"] = false,
		["Id"] = "",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["Settings"] = {
		["ButtonPositions"] = {
			["Unreliable"] = false,
			["Id"] = "2",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Create"] = {
			["Unreliable"] = false,
			["Id"] = "0",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Update"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "1",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "1",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
	},
	["OnServerError"] = {
		["Unreliable"] = false,
		["Id"] = "\n",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["Command"] = {
		["Unreliable"] = false,
		["Id"] = "",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["DeathScreen"] = {
		["Unreliable"] = false,
		["Id"] = "d",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["Quest"] = {
		["Unreliable"] = false,
		["Id"] = "",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["AFK"] = {
		["Unreliable"] = false,
		["Id"] = "",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["CharacterRemoving"] = {
		["Unreliable"] = false,
		["Id"] = "^",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["Loaded"] = {
		["Unreliable"] = false,
		["ClientEvent"] = {
			["Id"] = "e",
			["Fire"] = function() end,
			["On"] = function() end,
			["Unreliable"] = false,
		},
		["Id"] = "e",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["DoorLocked"] = {
		["Unreliable"] = false,
		["Id"] = "",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["Misc"] = {
		["BatHit"] = {
			["Unreliable"] = false,
			["Id"] = ">",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Ragdoll"] = {
			["Unreliable"] = false,
			["Id"] = "J",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["UseOreoBar"] = {
			["Unreliable"] = false,
			["Id"] = "A",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Flashlight"] = {
			["Unreliable"] = false,
			["Id"] = "?",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Dismiss"] = {
			["Unreliable"] = false,
			["Id"] = "=",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["CameraToggle"] = {
			["Unreliable"] = false,
			["Id"] = "8",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["CreateTitle"] = {
			["Unreliable"] = false,
			["Id"] = "I",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["TweenItem"] = {
			["Unreliable"] = false,
			["Id"] = "9",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["TeleportPlayer"] = {
			["Unreliable"] = false,
			["Id"] = "C",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["KillEffect"] = {
			["Unreliable"] = false,
			["Id"] = ";",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["HealPotion"] = {
			["Unreliable"] = false,
			["Id"] = "G",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["StaminaChange"] = {
			["Id"] = "F",
			["FireClients"] = function() end,
			["OnClient"] = function() end,
			["FireFilteredClients"] = function() end,
			["FireClient"] = function() end,
			["OnServer"] = function() end,
			["FireAllClientsExcept"] = function() end,
			["FireServer"] = function() end,
			["FireAllClients"] = function() end,
			["CallMode"] = "Signal",
			["Unreliable"] = false,
			["Signal"] = {
			},
		},
		["ToggleMovement"] = {
			["Unreliable"] = false,
			["Id"] = "D",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["OnPurchase"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "H",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "H",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["ToolbarVisibility"] = {
			["Unreliable"] = false,
			["Id"] = "E",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Input"] = {
			["Id"] = "<",
			["FireClients"] = function() end,
			["OnClient"] = function() end,
			["FireFilteredClients"] = function() end,
			["FireClient"] = function() end,
			["OnServer"] = function() end,
			["FireAllClientsExcept"] = function() end,
			["FireServer"] = function() end,
			["FireAllClients"] = function() end,
			["CallMode"] = "Signal",
			["Unreliable"] = false,
			["Signal"] = {
			},
		},
		["UseHealPotion"] = {
			["Unreliable"] = false,
			["Id"] = "@",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Polaroid"] = {
			["Unreliable"] = false,
			["Id"] = "B",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Effect"] = {
			["Unreliable"] = false,
			["Id"] = ":",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
	},
	["Teacher"] = {
		["Enrage"] = {
			["Unreliable"] = false,
			["Id"] = "",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["AmmoUpdate"] = {
			["Unreliable"] = false,
			["Id"] = "",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["InstDoorOpen"] = {
			["Unreliable"] = false,
			["Id"] = "-",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["AbilityUI_Update"] = {
			["Id"] = "\"",
			["FireClients"] = function() end,
			["OnClient"] = function() end,
			["FireFilteredClients"] = function() end,
			["FireClient"] = function() end,
			["OnServer"] = function() end,
			["FireAllClientsExcept"] = function() end,
			["FireServer"] = function() end,
			["FireAllClients"] = function() end,
			["CallMode"] = "Signal",
			["Unreliable"] = false,
			["Signal"] = {
			},
		},
		["UpdatePounceState"] = {
			["Unreliable"] = false,
			["Id"] = "(",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["UpdateChargeState"] = {
			["Unreliable"] = false,
			["Id"] = ")",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["EnragedVisibility"] = {
			["Unreliable"] = false,
			["Id"] = "#",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["BloomieBar"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "&",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "&",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Instinct"] = {
			["Unreliable"] = false,
			["Id"] = ",",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Tips"] = {
			["Unreliable"] = false,
			["Id"] = " ",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["PounceFail"] = {
			["Unreliable"] = false,
			["Id"] = "+",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["PounceHit"] = {
			["Unreliable"] = false,
			["Id"] = "*",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Blocked"] = {
			["Unreliable"] = false,
			["Id"] = "",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["ReplicateBlade"] = {
			["Unreliable"] = false,
			["Id"] = "'",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["ReturnPrompt"] = {
			["Unreliable"] = false,
			["Id"] = "%",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["ChargeRegister"] = {
			["Unreliable"] = false,
			["Id"] = "$",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["AbilityCooldown"] = {
			["Id"] = "!",
			["FireClients"] = function() end,
			["OnClient"] = function() end,
			["FireFilteredClients"] = function() end,
			["FireClient"] = function() end,
			["OnServer"] = function() end,
			["FireAllClientsExcept"] = function() end,
			["FireServer"] = function() end,
			["FireAllClients"] = function() end,
			["CallMode"] = "Signal",
			["Unreliable"] = false,
			["Signal"] = {
			},
		},
	},
	["GiftEvent"] = {
		["Unreliable"] = false,
		["ClientEvent"] = {
			["Id"] = "a",
			["Fire"] = function() end,
			["On"] = function() end,
			["Unreliable"] = false,
		},
		["Id"] = "a",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["Shop"] = {
		["Sell"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "7",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "7",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Buy"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "6",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "6",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
	},
	["CharacterAdded"] = {
		["Unreliable"] = false,
		["Id"] = "]",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["SetLookAngles"] = {
		["Unreliable"] = false,
		["ClientEvent"] = {
			["Id"] = "c",
			["Fire"] = function() end,
			["On"] = function() end,
			["Unreliable"] = false,
		},
		["Id"] = "c",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["AdminPanelClient"] = {
		["Unreliable"] = false,
		["Id"] = "",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["AllLights"] = {
		["Unreliable"] = false,
		["Id"] = "\\",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["Alice"] = {
		["Cutscene"] = {
			["Unreliable"] = false,
			["Id"] = "",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Jumpscare"] = {
			["Unreliable"] = false,
			["Id"] = "",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Phase2"] = {
			["TentacleGround"] = {
				["Unreliable"] = false,
				["Id"] = "",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
			["DoorCrouch"] = {
				["Unreliable"] = false,
				["Id"] = "",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
			["Spikes"] = {
				["Id"] = "",
				["FireClients"] = function() end,
				["OnClient"] = function() end,
				["FireFilteredClients"] = function() end,
				["FireClient"] = function() end,
				["OnServer"] = function() end,
				["FireAllClientsExcept"] = function() end,
				["FireServer"] = function() end,
				["FireAllClients"] = function() end,
				["CallMode"] = "Signal",
				["Unreliable"] = false,
				["Signal"] = {
				},
			},
			["SpiderCrawl"] = {
				["Unreliable"] = false,
				["Id"] = "",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
			["UpdateTentaclePosition"] = {
				["Unreliable"] = false,
				["Id"] = "",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
			["IntroFinished"] = {
				["Unreliable"] = false,
				["Id"] = "",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
			["DemonVision"] = {
				["Unreliable"] = false,
				["Id"] = "",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
		},
		["InsideAliceRoom"] = {
			["Unreliable"] = false,
			["Id"] = "",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Phase1"] = {
			["Teleport"] = {
				["Unreliable"] = false,
				["Id"] = "\n",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
			["ChargeGrab"] = {
				["Unreliable"] = false,
				["Id"] = "",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
			["SpikeR"] = {
				["Unreliable"] = false,
				["Id"] = "",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
			["Retreat"] = {
				["Unreliable"] = false,
				["Id"] = "",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
			["SpikeT"] = {
				["Unreliable"] = false,
				["Id"] = "",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
			["GrabVictim"] = {
				["Unreliable"] = false,
				["Id"] = "",
				["Client"] = function() end,
				["Validate"] = function() end,
				["Server"] = function() end,
			},
		},
		["CreateUI"] = {
			["Unreliable"] = false,
			["Id"] = "",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Seen"] = {
			["Unreliable"] = false,
			["Id"] = "",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
	},
	["Skins"] = {
		["Equip"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "5",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "5",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Purchase"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "3",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "3",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
		["Update"] = {
			["Unreliable"] = false,
			["ClientEvent"] = {
				["Id"] = "4",
				["Fire"] = function() end,
				["On"] = function() end,
				["Unreliable"] = false,
			},
			["Id"] = "4",
			["Client"] = function() end,
			["Validate"] = function() end,
			["Server"] = function() end,
		},
	},
	["Hitboxes"] = {
		["State"] = {
			["Id"] = ".",
			["FireClients"] = function() end,
			["OnClient"] = function() end,
			["FireFilteredClients"] = function() end,
			["FireClient"] = function() end,
			["OnServer"] = function() end,
			["FireAllClientsExcept"] = function() end,
			["FireServer"] = function() end,
			["FireAllClients"] = function() end,
			["CallMode"] = "Signal",
			["Unreliable"] = false,
			["Signal"] = {
			},
		},
		["OnHit"] = {
			["Id"] = "/",
			["FireClients"] = function() end,
			["OnClient"] = function() end,
			["FireFilteredClients"] = function() end,
			["FireClient"] = function() end,
			["OnServer"] = function() end,
			["FireAllClientsExcept"] = function() end,
			["FireServer"] = function() end,
			["FireAllClients"] = function() end,
			["CallMode"] = "Signal",
			["Unreliable"] = false,
			["Signal"] = {
			},
		},
	},
	["ThrowLibraryBook"] = {
		["Unreliable"] = false,
		["Id"] = "b",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
	["SkinModelLoader"] = {
		["Unreliable"] = false,
		["ClientEvent"] = {
			["Id"] = "f",
			["Fire"] = function() end,
			["On"] = function() end,
			["Unreliable"] = false,
		},
		["Id"] = "f",
		["Client"] = function() end,
		["Validate"] = function() end,
		["Server"] = function() end,
	},
}